### PR TITLE
mikmod, a module player

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
mikmod2
mikmod-3 2 8

File: media-sound/mikmod/mikmod-3.2.8.recipe [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	mikmod$secondaryArchSuffix = $portVersion
+	cmd:mikmod$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
```
Comment (diversys) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: You can omit brackets here

Comment (Begasus) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: Ah, probably a remainder of the libmikmod recipe (probably in there too then) ;)

File: media-sound/mikmod/mikmod-3.2.8.recipe [hunk 2]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="A module player for Unix"
+DESCRIPTION="libmikmod supports many formats, including mod, s3m, it, and xm."
```
Comment (korli) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: The description of libmikmod doesn't fit with mikmod :)

File: media-sound/mikmod/mikmod-3.2.8.recipe [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+HOMEPAGE="http://mikmod.sourceforge.net"
+COPYRIGHT="1998-2017 Jean-Paul Mikkers, Jake Stine, Steve McIntyre,
+Frank Loemker, Andrew Zabolotny."
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://sf.net/projects/mikmod/files/mikmod/$portVersion/mikmod-$portVersion.tar.gz"
+CHECKSUM_SHA256="dbb01bc36797ce25ffcab2b3bf625537b85b42534344e1808236ca612fbaa4cc"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```
Comment (korli) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: You can probably drop SECONDARY_ARCHITECTURES and $secondaryArchSuffix

File: media-sound/mikmod/mikmod-3.2.8.recipe (hunk 2 above)
Comment (Begasus) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: whoops :) missed that one :) Aside from that most of the description is the same :)

File: media-sound/mikmod/mikmod-3.2.8.recipe (hunk 3 above)
Comment (Begasus) [inline media-sound/mikmod/mikmod-3.2.8.recipe]: Ok, left it in to check on gcc2, will fix :)

**Pull request merged**